ext/mysqli/mysqli.stub.php

```diff
@@ -4,8 +4,10 @@
 
 final class mysqli_driver
 {
+    /** @deprecated 8.1.0 */
```

Comment:
Generally, something is only deprecated after the new preferred usage is added. See https://github.com/php/php-src/pull/6754 for a similar example.

Reply:
We could do that too. Generally, I agree that there should be first an alternate way and then deprecation, but in this case there already is an alternate way. If you think that this is a good idea then we can deprecate `mysqli::get_client_info()` now and deprecate properties in PHP 8.2. Not sure it will make much difference here. I just didn't want to add a fifth way of accessing these values without clearly marking which ones will be made obsolete.

Reply:
To be honest, I don't have any strong preference, because your reasoning makes sense for me.